app/main.development.js

```diff
@@ -885,3 +886,5 @@ app.on("before-quit", async (event) => {
   }
   app.exit(0);
 });
+
+handle("do-with-ledger", doWithLedger);
```

Comment:
I guess will need something in `externalRequests.js` for production to use this gateway?